vaadinfrontend/src/main/java/life/qbic/datamanager/views/project/create/CreateProjectLayout.java

```diff
@@ -47,6 +50,10 @@ protected VerticalLayout initContent() {
 
     FormLayout formLayout = new FormLayout();
     formLayout.addFormItem(titleField, "Project Title");
+    formLayout.addFormItem(projectObjective, "Project Objective");
+    formLayout.setResponsiveSteps(new ResponsiveStep("200px", 1));
```

Comment:
As I just tackled the same issue just a short finding:
The min width is where you want the layout to behave like having the column count you specify.
As we (currently) only want one column independent of width, I would suggest to change this line
```suggestion
    formLayout.setResponsiveSteps(new ResponsiveStep("0", 1));
```
reference: https://vaadin.com/docs/latest/components/form-layout#sectioning